src/GamCommands.txt

```diff
@@ -380,13 +380,16 @@ Named items
 	is2svenforced|
 	is2svenrolled|
 	lastlogintime|
+	locations|location|
 	noneditablealiases|aliases|nicknames|
 	notes|note|
 	organizations|organization|
 	orgunitpath|org|ou|
 	phones|phone|
+	posixaccounts|posix|
 	primaryemail|username|
 	relations|relation|
+	ssh|sshkeys|sshpublickeys|
```

Comment:
you could probably simplify this doc file a lot by not listing every possible argument that performs the same actions and instead listing the "best" choice.